website/templates/profile/notifications.mako

```diff
@@ -48,13 +47,17 @@
             </div>
     </div>
 </div>
+</%def>
 
-<script type="text/javascript">
-
-    $script(['/static/js/notificationsConfig.js']);
-    $script.ready('NotificationsConfig', function() {
-        var notifications = new NotificationsConfig('#selectLists', '${website.settings.MAILCHIMP_GENERAL_LIST}');
-    });
-</script>
-
+<%def name="javascript()">
+    <% import website %>
+    <script type="text/javascript">
+        ${parent.javascript()}
```

Comment:
This should be just under the `def`, not within a `script` tag. Otherwise you might be putting the parent's `script` tags within a `script` tag.